cloud/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@wowarenalogs/cloud-functions",
```

Comment:
i constantly misremember the differences between package name and the folder name :D better to just make it @woarenalogs/cloud to be consistent with the folder structure?